### ISSUE TITLE
Add ElasticNet

### ISSUE THREAD

**Issue by Thilakraj1998** [blobcity/autoai#67]
Add ElasticNet model into the library.

Primary File to Change: https://github.com/blobcity/autoai/blob/main/blobcity/config/regressor_config.py

Reference ElasticNet Implementation:
https://github.com/blobcity/ai-seed/blob/main/Regression/Linear%20Models/ElasticNet.ipynb

Official API Reference for Parameter:
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.ElasticNet.html

Dependencies if any, must be appropriately added. Test run of the train function on a regression problem must pass, and the function must attempt to train an ElasticNet as a potential best fit model.
